src/query/compiler.js

```diff
@@ -211,6 +213,14 @@ assign(QueryCompiler.prototype, {
     );
   },
 
+  // Add comments to the query
+  comments() {
+    if (!this.queryComments.length) return '';
+    return this.queryComments
+      .map((comment) => `/* ${comment.comment} */`)
```

Comment:
Pretty dangerous, since this is not escaped in anyways so */ in middle of string will open query to sql injection... Probably would be good to mention in docs, that this method is very much vulnerable and should never be allowed to be read from user input.

Reply:
isnt this handled in [the proposed change to builder.js line 162](https://github.com/knex/knex/pull/2815/files#diff-17127b37f866fedbb3d1739324bd780bdb159055397cd2b702906fa5c02de974R162) ?